package.json

```diff
@@ -71,6 +71,7 @@
     "@storybook/builder-webpack5": "^6.5.14",
     "@storybook/manager-webpack5": "^6.5.14",
     "@storybook/react": "^6.4.22",
+    "@testing-library/dom": "^9.3.1",
```

Comment:
We're using this directly to import `within`

https://testing-library.com/docs/dom-testing-library/api-within/